package/setup.py

```diff
@@ -24,11 +24,6 @@
 with open("requirements.txt", "r", encoding="utf-8") as f:
     requires = [x.strip() for x in f if x.strip()]
 
-# get test dependencies and installs
-with open("test_requirements.txt", "r", encoding="utf-8") as f:
-    test_requires = [x.strip() for x in f if x.strip() and not x.startswith("-r")]
-
-# Get the long description from the README file
```

Comment:
Seems this comment should still be present?

Reply:
I think the code is plenty readable without the comment.  Comment adds no value, so may as well remove it.